virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -72,6 +72,24 @@ The SVVP Certification applies to:
 
 //CNV-28720 Release note: CHANGE
 
+//CNV-28724 Release note: storage profile defaults
```

Comment:
It looks like a commit from one of your other branches got into this PR. That usually happens if you forget to switch to main before creating a new branch. Easiest solution is probably to close this PR and create a new branch. Or else see whether you can delete this section and squash your commits, but the original commit might not be squashable.

Reply:
I hope it's okay to leave as is. The Add table commit was not something I wanted to keep in the other PR.